### PR TITLE
📝 A take on 3 summary bullets

### PR DESCRIPTION
**Pull request by falexwolf**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,13 +3,12 @@
...
 ```
 
-Organization of biological knowledge and metadata curation.
+- Look up terms with auto-complete.
+- Map and curate metadata.
+- Manage public & custom ontologies and their versions.
 
-- Lookup & curate metadata based on scientific standards.
-- Access biological knowledge without the timeouts of many existing REST endpoints.
-- Extend Bionty with custom ontologies or terms.
```
Comment (falexwolf) [inline docs/index.md:10]: Still confused about the ability of adding single terms outside of SQL!

I think there should be one and only one way to store persistently custom knowledge (sets of terms that extend ontologies).

[And again: plugging in entire custom ontologies is absolutely within scope for me for Bionty, but entirely different functionality.]

Comment (Zethson) [reply docs/index.md:10]: See also https://github.com/laminlabs/bionty/issues/222

File: docs/index.md [hunk 2]
```diff
@@ -3,13 +3,12 @@
 :end-line: 5
 ```
 
-Organization of biological knowledge and metadata curation.
+- Look up records with auto-complete.
```
Comment (falexwolf) [inline docs/index.md:6]: I had "terms" here, but then realized there are also IDs, and potentially other fields of records. So, maybe "records" is good.

[This also makes me think about the auto-complete feature for data-driven entities within LaminDB.]

File: docs/index.md (hunk 1 above)
Comment (falexwolf) [reply docs/index.md:10]: OK! I'm  pretty happy about append-to DataFrame! And right, you linked this before! I'll add another thought there.

**Pull request merged**